best-practices/releasing_gems.md

```diff
@@ -26,6 +30,10 @@ $ gem owner base_indexer -a sul-devops-team@lists.stanford.edu
 
 The service account will ensure we all have access to our collective work. 
 
+### Run the gem bulk update script
+
+As of late 2018, we use a [script](https://github.com/sul-dlss/infrastructure-update-scripts/blob/master/grant_revoke_gem_authority.rb) (adapted from a Samvera community script) to ensure all Infrastructure and Access team members can release gems to rubygems.org for all gems in the DLSS portfolio. Run this script after you push a gem for the first time to provide access to fellow DLSS developers. See the script for the prerequisite steps. If there are any errors in the output, discuss them in an appropriate channel (DLSS developers listserv, `#software-developers` Slack channel, etc.).
```

Comment:
I think it's great that this discussion has been sparked. My two cents. 

While i think its great that people be given the permissions to release and maintain the software that is part of their job, my hesitancy to 👍 this stems from the fact that we are just giving everyone permission to everything without consideration or perhaps training about what implications of owning a gem may be, or what the community norms around that gem might be (even though it lives in sul-dlss).

I'd be in favor of the service account approach so that there is shared ownership rather than granting everyone owner on everything. Obviously, this has been flawed since there wan't understanding on who had access to it. Ideally the service account way would prompts the discussion to happen about who is maintaining what projects, and privileges can be granted when need.

I'm not 👎 just... 👌 . But wanted to share my concern. I think all of the automation around maintenance is a great thing.

Reply:
One last thing, we should collectively do a better job as a department on this. And I just wanted to thank y'all for working on resolving these issues.